xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.h

```diff
@@ -110,6 +114,21 @@ namespace OVERLAY {
     void SetVideoRect(CRect &source, CRect &dest, CRect &view);
     void SetStereoMode(const std::string &stereomode);
 
+    /*! \brief Set the subtitle position, it depends on current screen resolution
+     *  \param value The subtitle position in pixels
+     *  \param save The value will be save to resolution info
+     */
+    void SetSubtitlePosition(const double& value, bool save);
```

Comment:
Would the term "vertical shift" be more accurate than "position"?

Reply:
this in reality set the subtitle vertical position not the shift difference
i can rename it like `SetSubtitleVerticalPosition` / `GetSubtitleVerticalPosition`